### PR TITLE
indentation and statusCode

### PR DESCRIPTION
**Pull request by vchepeli**

### CODE REVIEW

File: lib/routes/user.js [hunk 1]
```diff
@@ -5,36 +5,39 @@ var cors = require('cors');
...
       res.json(statusObj);
       statusObj.clearStatus();
-    }else if(statusObj.isInProgress()){
+    } else if (statusObj.isInProgress()) {
+      res.statusCode = 255;
       res.json(statusObj);
-    }else{
+    } else {
       insertDataIntoDb(req.query.numberOfRecords);
+      res.statusCode = 255;
```
Comment (wtrocki) [inline lib/routes/user.js]: 255 is invalid status code (I do not know that status code). 
I think that proper status core here will be 202.

File: lib/routes/user.js [hunk 2]
```diff
@@ -5,36 +5,39 @@ var cors = require('cors');
...
-  user.get('/generate', function(req, res){
-    if(statusObj.isFinished()){
+  user.get('/generate', function (req, res) {
+    if (statusObj.isFinished()) {
+      res.statusCode = 200;
       res.json(statusObj);
       statusObj.clearStatus();
-    }else if(statusObj.isInProgress()){
+    } else if (statusObj.isInProgress()) {
+      res.statusCode = 255;
```
Comment (wtrocki) [inline lib/routes/user.js]: Proper status code here may be 200 or 409

File: lib/routes/user.js [hunk 3]
```diff
@@ -5,36 +5,39 @@ var cors = require('cors');
...
-  user.get('/', function(req, res){
+  user.get('/', function (req, res) {
     res.json(userService.getRandomUser());
   });
 
-  user.get('/generate', function(req, res){
-    if(statusObj.isFinished()){
+  user.get('/generate', function (req, res) {
+    if (statusObj.isFinished()) {
+      res.statusCode = 200;
```
Comment (wtrocki) [inline lib/routes/user.js:19]: Proper status code here may be 200 or 409

File: Gruntfile.js [hunk 4]
```diff
@@ -47,7 +47,7 @@ module.exports = function(grunt) {
       // environment variables - see https://github.com/jsoverson/grunt-env for more information
       local: {
         FH_USE_LOCAL_DB: true,
-        FH_SERVICE_MAP: function() {
+        FH_SERVICE_MAP: function () {
```
Comment (wtrocki) [inline Gruntfile.js]: This project was already formatted. This looks like standard formatting profile in Webstorm, however I would not use it as default profile for opensource project. I would recommend to adjust profile or format changes only for blocks of code that were changed (by selecting them).

Comment (ghost) [inline Gruntfile.js]: Ok np. will revert changes probably formatting does not make sense here

**Comment by wtrocki** [comment]
Currently logic for status can check startDate and endDate. 
To simplify that on various clients we can just introduce another json field that will reflect status: [inprogress | finished | started] or use some http verbs as in comments. I will accept everything as long we will use some valid http1.1 statuses. Currently this may cause some clients to fail.

**Pull request merged**